src/cdk/a11y/focus-trap/configurable-focus-trap.spec.ts

```diff
@@ -0,0 +1,196 @@
+import {AfterViewInit, Component, ElementRef, ViewChild} from '@angular/core';
+import {async, ComponentFixture, fakeAsync, flush, TestBed} from '@angular/core/testing';
+import {A11yModule, ConfigurableFocusTrapFactory, ConfigurableFocusTrap} from '../index';
+
+
+describe('ConfigurableFocusTrap', () => {
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [A11yModule],
+      declarations: [
+        NestedFocusTraps,
+        SimpleFocusTrap,
+      ],
+    });
+
+    TestBed.compileComponents();
+  }));
+
+  describe('with EventListenerFocusTrapInertStrategy', () => {
+    let fixture: ComponentFixture<SimpleFocusTrap>;
+    let componentInstance: SimpleFocusTrap;
+    let focusTrapInstance: ConfigurableFocusTrap;
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(SimpleFocusTrap);
```

Comment:
Having it in a `beforeEach` like this makes it difficult to add new test cases that use a different template. I've been trying to move the tests away to a pattern where we call `createComponent` inside the test itself. You can see an example in `cdk/table.spec.ts`.

Reply:
Cool! Changed.